src/graph/graph_hub.c

```diff
@@ -6,92 +6,6 @@
 
 #include "graph_hub.h"
 #include "../query_ctx.h"
-#include "../undo_log/undo_log.h"
-
-// delete all references to a node from any relevant index
-static void _DeleteNodeFromIndices
-(
-	GraphContext *gc,
-	Node *n
-) {
-	ASSERT(n  != NULL);
-	ASSERT(gc != NULL);
-
-	Schema   *s      = NULL;
-	Graph    *g      = gc->g;
-	EntityID node_id = ENTITY_GET_ID(n);
-
-	// retrieve node labels
-	uint label_count;
-	NODE_GET_LABELS(g, n, label_count);
-
-	for(uint i = 0; i < label_count; i++) {
-		int label_id = labels[i];
-		s = GraphContext_GetSchemaByID(gc, label_id, SCHEMA_NODE);
-		ASSERT(s != NULL);
-
-		// update any indices this entity is represented in
-		Schema_RemoveNodeFromIndex(s, n);
-	}
-}
-
-static void _DeleteEdgeFromIndices
-(
-	GraphContext *gc,
-	Edge *e
-) {
-	Schema *s = NULL;
-	Graph  *g = gc->g;
-
-	int relation_id = Edge_GetRelationID(e);
-
-	s = GraphContext_GetSchemaByID(gc, relation_id, SCHEMA_EDGE);
-
-	// update any indices this entity is represented in
-	Schema_RemoveEdgeFromIndex(s, e);
-}
-
-// add node to any relevant index
-static void _AddNodeToIndices
-(
-	GraphContext *gc,
-	Node *n
-) {
-	ASSERT(n  != NULL);
-	ASSERT(gc != NULL);
-
-	Schema   *s      = NULL;
-	Graph    *g      = gc->g;
-	EntityID node_id = ENTITY_GET_ID(n);
-
-	// retrieve node labels
-	uint label_count;
-	NODE_GET_LABELS(g, n, label_count);
```

Comment:
I see we used to call this macro in both `_AddNodeToIndices` and `_DeleteNodeFromIndices`
so basically in both add node to index and delete node from index, while the new API only requires us to provide the labels in only one of these operations